libraries/node-core-library/src/Path.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+
+/**
+ * Helper functions for working with file paths.
+ * @public
+ */
+export class Path {
```

Comment:
>Path [](start = 13, length = 4)

Is this confusing when we already use the "path" module ?

Reply:
I guess this is OK

---
In reply to: [169521201](https://github.com/Microsoft/web-build-tools/pull/537#discussion_r169521201) [](ancestors = 169521201)

Reply:
We discussed this and decided that Path should ultimately be a replacement for the low level path, similar to how fs-extra is a replacement for fs.

---
In reply to: [169521201](https://github.com/Microsoft/web-build-tools/pull/537#discussion_r169521201) [](ancestors = 169521201)

Reply:
Fixed